### PR TITLE
PC-18021 | add acquia/http-hmac-ruby repository into the Cortex

### PR DESCRIPTION
**Pull request by hashmath-acquia**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

**Review by blkperl — APPROVED** [review]

File: .acquia/.cortex/cortex.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  x-cortex-git:
+    github:
+      repository: acquia/http-hmac-ruby
+  x-cortex-owners:
+    - type: GROUP
+      name: acquia/cloud-cde
+      provider: GITHUB
+      description: acquia cde team
+    - type: SLACK
+      channel: cloud-eng-cde-alerts
```
Comment (nalakafernando) [inline .acquia/.cortex/cortex.yaml]: this can be `cloud-eng-platform-classic-alerts`

File: .acquia/.cortex/cortex.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  description: Enhancing web app and API security through Acquia HTTP HMAC Spec implementation.
+  x-cortex-tag: acquia/http-hmac-ruby
+  x-cortex-git:
+    github:
+      repository: acquia/http-hmac-ruby
+  x-cortex-owners:
+    - type: GROUP
+      name: acquia/cloud-cde
+      provider: GITHUB
+      description: acquia cde team
```
Comment (nalakafernando) [inline .acquia/.cortex/cortex.yaml]: Lets change to platform classic team

**Review by nalakafernando — COMMENTED** [review]
Once merged, check if this is available in https://cortex.acquia.io/admin/index

File: .acquia/.cortex/cortex.yaml (hunk 1 above)
Comment (hashmath-acquia) [reply .acquia/.cortex/cortex.yaml]: updated

File: .acquia/.cortex/cortex.yaml (hunk 2 above)
Comment (hashmath-acquia) [reply .acquia/.cortex/cortex.yaml]: updated

**Review by nalakafernando — APPROVED** [review]

**Pull request merged**